projects/schematics/src/migrations/4_0/rename-symbol/rename-symbol.ts

```diff
@@ -119,6 +173,296 @@ export const RENAMED_SYMBOLS_DATA: RenamedSymbol[] = [
     newNode: CLEAR_CHECKOUT_FACADE,
     newImportPath: SPARTACUS_CHECKOUT_ROOT,
   },
+  // projects/storefrontlib/src/cms-components/misc/qualtrics/qualtrics-loader.service.ts
+  {
+    previousNode: QUALTRICS_LOADER_SERVICE,
+    previousImportPath: SPARTACUS_STOREFRONTLIB,
+    newImportPath: `${SPARTACUS_QUALTRICS}/components`,
+  },
+  // projects/storefrontlib/src/cms-components/misc/qualtrics/config/qualtrics-config.ts
+  {
+    previousNode: QUALTRICS_CONFIG,
+    previousImportPath: SPARTACUS_STOREFRONTLIB,
+    newImportPath: `${SPARTACUS_QUALTRICS}/components`,
+  },
+  // projects/storefrontlib/src/cms-components/misc/qualtrics/qualtrics-loader.service.ts
+  {
+    previousNode: QUALTRICS_EVENT_NAME,
+    previousImportPath: SPARTACUS_STOREFRONTLIB,
+    newImportPath: `${SPARTACUS_QUALTRICS}/components`,
+  },
+  // projects/storefrontlib/src/cms-components/misc/qualtrics/qualtrics.component.ts
+  {
+    previousNode: QUALTRICS_COMPONENT,
+    previousImportPath: SPARTACUS_STOREFRONTLIB,
+    newImportPath: `${SPARTACUS_QUALTRICS}/components`,
+  },
+  // projects/storefrontlib/src/cms-components/misc/qualtrics/qualtrics.module.ts
+  {
+    previousNode: QUALTRICS_MODULE,
+    previousImportPath: SPARTACUS_STOREFRONTLIB,
+    newNode: 'QualtricsComponentsModule',
+    newImportPath: `${SPARTACUS_QUALTRICS}/components`,
+  },
+  // projects/storefrontlib/src/cms-components/asm/asm.module.ts
+  {
+    previousNode: ASM_MODULE,
+    previousImportPath: SPARTACUS_STOREFRONTLIB,
+    newNode: 'AsmComponentsModule',
+    newImportPath: `${SPARTACUS_ASM}/components`,
+  },
+  // projects/core/src/asm/asm.module.ts
+  {
+    previousNode: ASM_MODULE,
+    previousImportPath: SPARTACUS_CORE,
+    newImportPath: `${SPARTACUS_ASM}/core`,
+  },
```

Comment:
Now I think we should keep this deprecation as a comment. This module was split functionally into 2: AsmCoreModule and AsmRootModule